app/models/manageiq/providers/openstack/provider.rb

```diff
@@ -1,11 +1,11 @@
-class ProviderOpenstack < Provider
+class ManageIQ::Providers::Openstack::Provider < ::Provider
```

Comment:
I think the issue is that if the `openstack.rb` file lives in `manageiq/providers/`, then you cannot extract the OpenStack provider in `manageiq/providers/openstack/` into its own repo.